### ISSUE TITLE
Crash after importing single asteroid

### ISSUE THREAD

**Issue by jl2008** [Stellarium/stellarium#1970]
Stellarium 0.21.2, 64-bit, running on Windows 10 Home Version 10.0.19043.

Steps:
1. Start Stellarium
2. F2, Plugins, Solar System Editor, Configure, Solar System, Import MPC, Online Search, type Pallas.
3. One result is returned, check, click on Add Object (leave the other settings to defaults: _Add new and update existing objects_; _update only the orbital elements_ checked.
4. Stellarium crashed.

Unfortunately I could not reproduce it, even after deleting Pallas from `ssystem_minor.ini` but maybe the info below helps nonetheless. If not, you can close it unless I manage to reproduce it - I did try with other asteroids but it did not reoccur.

```
Faulting application name: stellarium.exe, version: 0.21.2.0, time stamp: 0x6151f0dd
Faulting module name: stellarium.exe, version: 0.21.2.0, time stamp: 0x6151f0dd
Exception code: 0xc0000005
Fault offset: 0x00000000002aa498
Faulting process id: 0xb98
Faulting application start time: 0x01d7c303dfa805f5
Faulting application path: C:\Program Files\Stellarium\stellarium.exe
Faulting module path: C:\Program Files\Stellarium\stellarium.exe
Report Id: 2f339fed-c620-4811-af6e-2e56c8582f7d
Faulting package full name: 
Faulting package-relative application ID: 
```

```
Loaded plugin "SolarSystemEditor"
Using the ssystem_minor.ini file that already exists in the user directory...
Initializing planets GL shaders... 
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
  OpenType support missing for "Verdana", script 11
  OpenType support missing for "MS Shell Dlg 2", script 11
  OpenType support missing for "Arial", script 11
  OpenType support missing for "MS UI Gothic", script 11
  OpenType support missing for "SimSun", script 11
  OpenType support missing for "Verdana", script 12
  OpenType support missing for "MS Shell Dlg 2", script 12
  OpenType support missing for "Arial", script 12
  OpenType support missing for "MS UI Gothic", script 12
  OpenType support missing for "SimSun", script 12
  OpenType support missing for "Arial Unicode MS", script 12
  OpenType support missing for "Verdana", script 13
  OpenType support missing for "MS Shell Dlg 2", script 13
  OpenType support missing for "Arial", script 13
  OpenType support missing for "MS UI Gothic", script 13
  OpenType support missing for "SimSun", script 13
  OpenType support missing for "Verdana", script 14
  OpenType support missing for "MS Shell Dlg 2", script 14
  OpenType support missing for "Arial", script 14
  OpenType support missing for "MS UI Gothic", script 14
  OpenType support missing for "SimSun", script 14
  OpenType support missing for "Verdana", script 16
  OpenType support missing for "MS Shell Dlg 2", script 16
  OpenType support missing for "Arial", script 16
  OpenType support missing for "MS UI Gothic", script 16
  OpenType support missing for "SimSun", script 16
  OpenType support missing for "Verdana", script 17
  OpenType support missing for "MS Shell Dlg 2", script 17
  OpenType support missing for "Arial", script 17
  OpenType support missing for "MS UI Gothic", script 17
  OpenType support missing for "SimSun", script 17
  OpenType support missing for "Arial Unicode MS", script 17
  OpenType support missing for "Verdana", script 18
  OpenType support missing for "MS Shell Dlg 2", script 18
  OpenType support missing for "Arial", script 18
  OpenType support missing for "MS UI Gothic", script 18
  OpenType support missing for "SimSun", script 18
  OpenType support missing for "Verdana", script 19
  OpenType support missing for "MS Shell Dlg 2", script 19
  OpenType support missing for "Arial", script 19
  OpenType support missing for "MS UI Gothic", script 19
  OpenType support missing for "SimSun", script 19
  OpenType support missing for "Arial Unicode MS", script 19
  OpenType support missing for "Verdana", script 20
  OpenType support missing for "MS Shell Dlg 2", script 20
  OpenType support missing for "Arial", script 20
  OpenType support missing for "MS UI Gothic", script 20
  OpenType support missing for "SimSun", script 20
  OpenType support missing for "Arial Unicode MS", script 20
Done reading minor planet orbital elements. Recognized 1 candidate objects out of 1 lines.
appendToSolarSystemConfigurationFile begin ... 
Updating objects...
Updated successfully "2pallas"
Loading Solar System data (1: planets and moons) ...
Loading from : "./data/ssystem_major.ini"
SolarSystem has  83 entries.
Loaded 83 Solar System bodies
Loading Solar System data (2: minor bodies)...
Loading from : "C:\\Users\\jl\\AppData\\Roaming\\Stellarium/data/ssystem_minor.ini"
SolarSystem has  190 entries.
Loaded 107 Solar System bodies
File ssystem_minor.ini is loaded successfully...
Loading nomenclature for Solar system bodies ...
Loaded 15747 / 15749 items of planetary surface nomenclature
WARNING - The next planets to assign nomenclature items is not found: "Dactyl"

```

**Comment by gzotti** [comment]
This logfile looks OK. I must flag this as incomplete for now. If you find how to reproduce it, please tell us more.

**Comment by gzotti** [comment]
I am afraid most users don't even know about logfiles. How would they clean up manually?

**Comment by alex-w** [comment]
@jl2008 did you have selected Pallas right before updating the elements?

**Comment by jl2008** [comment]
Thanks for the update! Unfortunately I don't recall whether I had selected Pallas right before. It's quite possible if I quickly wanted to see where it roughly is, even before updating the elements to the latest.

I've just tried now with other asteroids, for one (Bennu) without selecting it first, for the other (2019 XS) with selecting it first. Stellarium did not crash in either scenario, though. If there is anything else you want me try, let me know. 

Thank you for all the work.
